packages/core/src/interpreter.ts

```diff
@@ -838,13 +842,24 @@ export class Interpreter<
             return;
           }
 
-          const source = serviceCreator(context, _event.data);
+          const actor = serviceCreator(context, _event.data, {
+            parent: this as any,
```

Comment:
do we need to expose the whole parent to the service creator? couldnt we limit this to a very limited minimum? What kind of properties/methods can be useful for `serviceCreator`?

Reply:
All we need is `parent.send` which is what the `Actor` interface will become (future PR)